src/python/bot/tasks/minimize_task.py

```diff
@@ -1122,25 +1145,15 @@ def _unset_dedup_flags():
         testcase_file_path, file_host.rebase_to_worker_root(testcase_file_path))
     rebased_output_file_path = file_host.rebase_to_worker_root(output_file_path)
 
-  arguments = environment.get_value('APP_ARGS', '')
-  arguments += (' --cf-{tool_name}-timeout={timeout} '
-                '--cf-{tool_name}-to={output_file_path}').format(
-                    tool_name=tool_name,
-                    output_file_path=rebased_output_file_path,
-                    timeout=timeout)
-  command = testcase_manager.get_command_line_for_application(
-      file_to_run=testcase_file_path,
-      app_args=arguments,
-      needs_http=testcase.http_flag)
-  logs.log('Executing command: %s' % command)
+  arguments = data_handler.get_arguments(testcase, strip_target=False).split()
```

Comment:
This strip_target feels a little hacky as it relies on location in arguments (maybe we stop using arguments altogether in future). Why not just call get_fuzzer_display().target since we have the testcase object here.